src/Input/TextField/TextField.tsx

```diff
@@ -135,19 +135,23 @@ class TextField extends React.Component<Props, State> {
   }
 }
 
-export type textFieldType = 'text' | 'password';
+export type textFieldType = 'text' | 'password' | 'number';
 
-interface Props extends React.ComponentPropsWithoutRef<typeof TextFieldInput> {
+export interface Props
+  extends React.ComponentPropsWithoutRef<typeof TextFieldInput> {
   type: textFieldType;
   label: string;
   disabled?: boolean;
   className?: string;
   removeFloatingLabel?: boolean;
+  min?: number;
+  max?: number;
+  step?: number;
```

Comment:
I am not sure whether we need `step`,  do you need it in your use case?

Reply:
I don't need it but I added it because this is supposed to be a replacement for `<input type="number />` which has it. Since this is a component library I figured it would be good to cover as many potential use cases as possible.

Reply:
got it